bin/bookkeeper

```diff
@@ -62,13 +58,25 @@ bookkeeper_help() {
     cat <<EOF
 Usage: bookkeeper <command>
 where command is one of:
+
+[service commands]
+
     bookie              Run a bookie server
     autorecovery        Run AutoRecovery service daemon
+    zookeeper           Run zookeeper server
+
+[local development commands]
+
     localbookie <n>     Run a test ensemble of <n> bookies locally
     standalone          Run a standalone cluster (with all service components) locally
```

Comment:
there is no simple way to hide this in shell script, or in order to do, it will make the script to complicated. so I leave the command in the script and have an error message to tell people to use 'localbookie <n>' instead of 'standalone'.